src/crypto.ts

```diff
@@ -1,3 +1,5 @@
+const { Blob } = require('blob-polyfill')
```

Comment:
Use `import { Blob } from 'blob-polyfill'` syntax -- it will be transpiled to require syntax.